### PR TITLE
[ADD] Seed admin to database

### PR DESCRIPTION
**Pull request by Japheth-Joepari**
Seeded admin to database: created the AdminSeeder file in seeders directory with the required fields and called the AdminSeeder::class in the databaseSeeder.php file in the seeders directory.

### CODE REVIEW

File: database/seeders/AdminSeeder.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        DB::table('users')->insert([
+            'first_name' => 'John',
+            'last_name' => 'Doe',
+            'username' => 'admin',
+            'email' => 'admin@example.com',
+            'phone_number' => null,
+            'address' => '123 Main Street',
+            'password' => Hash::make('password'), 
+            // 'confirm_password' => Hash::make('password'), //confirm_password column does does not exist
```
Comment (davydocsurg) [inline database/seeders/AdminSeeder.php:25]: Please, remove the comments

File: database/seeders/DatabaseSeeder.php [hunk 2]
```diff
@@ -18,5 +18,6 @@ public function run(): void
         //     'name' => 'Test User',
```
Comment (davydocsurg) [inline database/seeders/DatabaseSeeder.php:18]: Remove the comments here too

**Review by davydocsurg — CHANGES_REQUESTED** [review]

File: database/seeders/AdminSeeder.php (hunk 1 above)
Comment (Japheth-Joepari) [reply database/seeders/AdminSeeder.php]: Done.

File: database/seeders/DatabaseSeeder.php (hunk 2 above)
Comment (Japheth-Joepari) [reply database/seeders/DatabaseSeeder.php]: Done.

**Review by davydocsurg — APPROVED** [review]

**Pull request merged**